pkg/apis/pipeline/v1alpha1/run_types.go

```diff
@@ -36,11 +37,27 @@ var (
 	}
 )
 
+// EmbeddedRunSpec allows custom task definitions to be embedded
+type EmbeddedRunSpec struct {
+	runtime.TypeMeta `json:",inline,omitempty"`
```

Comment:
We consider this as a required field so do we need to add `omitempty` clause here?